crates/oxc_ast/src/ast/operator.rs

```diff
@@ -98,7 +98,15 @@ impl AssignmentOperator {
 
     #[must_use]
     pub fn is_bitwise(self) -> bool {
-        matches!(self, Self::BitwiseOR | Self::BitwiseXOR | Self::BitwiseAnd)
+        matches!(
+            self,
+            Self::BitwiseOR
+                | Self::BitwiseXOR
+                | Self::BitwiseAnd
+                | Self::ShiftLeft
```

Comment:
Eslint has those [test suits](https://github.com/eslint/eslint/blob/e39f28d8578a00f4da8d4ddad559547950128a0d/tests/lib/rules/no-bitwise.js#L47-L49), so I add more bitwises for Assignment.

Reply:
I fixed this on main.